.github/workflows/check-issue-has-release.yml

```diff
@@ -0,0 +1,42 @@
+name: Check for Release
+
+on:
+  pull_request:
+    branches:
+      - develop
+      - main
+      - 'feature/**'
+    types:
+      - opened
+      - closed
+      - reopened
+      - synchronize
+  pull_request_review:
+    types: [submitted]
+
+concurrency:
+  group: check-issue-has-release-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  check-issue-has-release:
+    name: Issue must have a ZenHub release
+    runs-on: ubuntu-latest
+    timeout-minutes: 20
+    if: false == ( ( github.event_name == 'pull_request' && ( github.event.action == 'closed' || github.event.pull_request.draft == true || contains( github.head_ref, 'dependabot/' ) ) ) || github.event.pull_request.head.repo.fork ) && github.event.review.state == 'approved'
+    steps:
+      - uses: actions/checkout@v3
+      - name: Setup Node.js (.nvmrc)
+        uses: actions/setup-node@v3
+        with:
+          node-version-file: '.nvmrc'
+          cache: 'npm'
+      - name: npm install
+        run: |
+          npm ci
+      - name: Check issue for release tag
+        run: |
+          ./bin/check-issue-has-release.sh
```

Comment:
Also, this file doesn't exist :)

```suggestion
          ./bin/check-issue-has-release.js
```

<img width="1047" alt="image" src="https://github.com/google/site-kit-wp/assets/1621608/56f04fe8-0d43-4763-a3f5-672e1a1a7fa6">